source/scss/_library/_objects.text.scss

```diff
@@ -377,10 +377,12 @@ a.o-kicker {
     line-height: 1.5;
   }
 
-  // ember compat
+  // ember compat <-- I see this is for Ember compatibility, but I had a bug
+  // regarding captions and this is what was breaking it. Perhaps there's
+  // another way to achieve compatibility - TJ
```

Comment:
ok, let's chat when I'm back in on wednesday. this has been tricky to get right, but I think we're close.

Reply:
Cool, sounds like a plan.